src/actions/edit-dl.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { useCallback, useMemo } from 'react';
+
+import { useModal } from '@zextras/carbonio-design-system';
+import { useTranslation } from 'react-i18next';
+import { DefaultTheme } from 'styled-components';
+
+import { EditDLControllerComponent } from '../components/edit-dl-controller';
+import { ACTION_IDS } from '../constants';
+import { DistributionList } from '../model/distribution-list';
+
+export type UIAction<ExecArg, CanExecArg> = {
+	id: string;
+	label: string;
+	icon: keyof DefaultTheme['icons'];
+	execute: (arg: ExecArg) => void;
+	canExecute: (arg: CanExecArg) => boolean;
```

Comment:
canExecute is never used, 
I suggest to try to use or remove

Reply:
See https://github.com/zextras/carbonio-contacts-ui/pull/169#discussion_r1438337223

Reply:
We actually forgot to use it